CreeDictionary/search_quality/sample.csv

```diff
@@ -1,70 +1,384 @@
+many snakes,kinêpikoskâw,,,,
+there are many snakes,kinêpikoskâw,,,,
+dances,nîmiw,nîmihitow,,,
+dancing time,nîminâniwan,nîmiw,,,
+go to bed,mâci-nipa,,,,
+Merry Christmas,miyo-manitowi-kîsikânisik,,,,
+merry christmas,miyo-manitowi-kîsikânisik,,,,
+i see you,kiwâpamitin,,,,
+he sees him,wâpamêw,,,,
+she sees him,wâpamêw,,,,
+starblanket,atâhkakohp,acâhkosa kâ-otakohpit,,,
+star blanket,atâhkakohp,acâhkosa kâ-otakohpit,,,
+being taught,kiskinwahamâkosiw,,,,
+they see us,niwâpamikonânak,kiwâpamikonawak,,,
+gym,mêtawêwikamik,,,,This item and below are from NS152
+lake,sâkahikan,,,,
+juice,sîwâpoy,,,,
+hospital,âhkosîwikamik,,,,
+sock,asikan,,,,
+pail,askihk,,,,
+potato,askipwâwi,,,,
+dog,atim,nitêm,,,very irregular inflections
+doll,awâsisihkân,,,,
+tobacco,ciscêmâs,cistêmâw,,,
+fork,cîscahikanis,,,,
+girl,iskwêsis,,,,
+woman,iskwêw,,,,
+fish,kinosêw,,,,
+bacon,kohkôsiwiyin,,,,
+corn,mahtâmin,,,,
+pencil,masinahikanâhtik,,,,
+slate,masinahikanâpisk,,,,
+pen,masinahikanâpiskos,,,,
+tea,maskihkîwâpoy,,,,
+bear,maskwa,,,,
+glass,minihkwâcikan,minihkwêwiyâkanis,,,
+cup,minihkwâcikanis,,,,
+cat,minôs,,,,
+pants,mitâs,,,,
+feather,mêkwan,,,,
+road,mêskanaw,,,,
+soup,mîcimâpoy,,,,
+bead,mîkis,,,,
+knife,môhkomân,,,,
+little knife,môhkomânis,,,,
+moose,môswa,,,,
+toboggan,napakâhtik,,,,
+frying pan,napwên,,,,
+goose,niska,,,,
+boy,nâpêsis,,,,
+man,nâpêw,,,,
+Cree person,nêhiyaw,,,,
+tomato,okiniy,,,,
+little carrot,oskâcâskos,,,,
+carrot,oskâtâsk,,,,
+nut,pakân,,,,
+pepper,papêskomina,,,,
+apple,picikwâs,,,,
+ball,pâkâhtowân,,,,
+sun,pîsim,,,,
+skunk,sikâk,,,,
+sugar,sîwinikan,,,,
+salt,sîwîhtâkan,,,,
+money,sôniyâw,,,,
+butter,tohtôsâpoy,,,,
+milk,tohtôsâpoy,,,,
+fur,wayân,,,,
+dish,wiyâkan,,,,
+banana,wâkâs,,,,
+rabbit,wâpos,,,,
+egg,wâwi,,,,
+little spoon,êmihkwânis,,,,
+brings it,pêtawêw,,,,
+store,atâwêwikamik,,,,
+pig,kohkôs,,,,
+shoe,maskisin,,,,
+horse,misatim,mistatim,,,
+cow,mostos,,,,
+restaurant,mîcisowikamik,,,,
+duck,sîsîp,,,,
+bank,sôniyâwikamik,,,,
+chair,têhtapiwin,,,,
+spoon,êmihkwân,,,,
+drinks,minihkwêw,,,,
+right,tâpwê,,,,
+puppy,acimosis,,,,
+land,askiy,,,,
+child,awâsis,,,,
+skin,masakay,,,,
+fingernail,maskasiy,,,,
+hand,micihciy,,,,
+finger,micihcîs,,,,
+nipple,micohcôsimis,,,,
+face,mihkwâkan,,,,
+ear,mihtawakay,,,,
+nose,mikot,,,,
+little foot,misicis,,,,
+foot,misit,,,,
+leg,miskât,,,,
+eye,miskîsik,,,,
+arm,mispiton,,,,
+head,mistikwân,,,,
+breast,mitohtôsim,,,,
+mouth,mitôn,,,,
+body,miyaw,,,,
+tooth,mîpit,,,,
+hair,mêscakâsa,,,,
+my foot,nisit,,,,
+my head,nistikwân,,,,
+who,awîna,,,,
+oh,ka,,,,
+again,kîhtwâm,,,,
+what,kîkwây,,,,
+but,mâka,,,,
+when,tânispî,,,,
+how,tânisîyisi,,,,
+where,tânitê,,,,
+why,tânêhki,,,,
+far away,wâhyaw,,,,
+soon,wîpac,,,,
+so,êkosi,,,,
+and,êkwa,mîna,,,
+son,nikosis,,,,
+aunt,nikâwîs,,,,
+mother,nikâwiy,,,,
+grandfather,nimosôm,,,,
+husband,ninâpêm,,,,
+parents,ninêkihikwak,,,,
+younger sibling,nisîmis,,,,
+daughter,nitânis,,,,
+friend,nitôtêm,,,,
+father,nôhtâwiy,,,,
+sibling,nîtisân,,,,
```

Comment:
sibling: include words for brother/sister, and include sibling for brother/sister search (note that Cree has "older brother","older sister", then "younger sibling" with no gender distinction, as well as the form given here which applies regardless of gender or relative age